src/veins/nodes/UnicastScenario.ned

```diff
@@ -0,0 +1,61 @@
+//
```

Comment:
This does not seem to be a general purpose scenario that is likely to be reused. Could we move this file into the _example/unicast_ directory?